meson.build

```
@@ -111,13 +111,18 @@ configure_file(
   configuration: cdata,
 )
 
+dl_dep = cc.find_library('dl', required: false)
+threads_dep = dependency('threads')
```

```suggestion
threads_dep = dependency('threads')
dl_dep = cc.find_library('dl', required: false)
```

(Nit: `threads` goes first since it's more generic/less specific than `dl`.)